client/src/i18n/fr.json

```diff
@@ -712,7 +713,7 @@
       "UNIT_WEIGHT"  : "Entrer le poids",
       "UNIT_VOLUME"  : "Entrer le volume",
       "USERNAME"     : "Entrer le nom utilisateur",
-      "VILLAGE"      : "Entrer village",
+      "VILLAGE"      : "Entrer village/Commune",
```

Comment:
This comment is not crucial for blocking this PR. 
`village/Commune` must be `Village/Commune` in uppercase